MainModule/Server/Core/Remote.lua

```diff
@@ -550,6 +550,7 @@ return function(Vargs, GetEnv)
 						local newenv = GetEnv(getfenv(),{
 							print = function(...) local nums = {...} for _,v in nums do Remote.Terminal.LiveOutput(p,"PRINT: "..tostring(v)) end end;
 							warn = function(...) local nums = {...} for _,v in nums do Remote.Terminal.LiveOutput(p,"WARN: "..tostring(v)) end end;
+							error = function(...) local nums = {...} for _,v in nums do Remote.Terminal.LiveOutput(p,"ERROR: "..tostring(v)) end end;
```

Comment:
are you sure this is necessary and/or works?

Reply:
wym